app/digests/api.py

```diff
@@ -37,6 +47,15 @@ def _create_response(self, data: Dict[str, Any]) -> Response:
     def _can_preview(self) -> bool:
         return self.request.META.get(settings.AUTHORIZATION_PREVIEW_HEADER, False)
 
+    def _publish_event(self) -> None:
+        try:
+            # could add a `DigestEvent` to `elife_bus_sdk`
+            # to replace the `Event` here, though functionality will not change.
+            LOGGER.info('_publish_event call for %s' % self.instance.id)
```

Comment:
global state is a smell, can we define `_publish_event(self, instance)` and pass `lambda: self._publish_event(instance)` to `transaction.on_commit()`?

Reply:
Its only stateful for the life of that object, `self.instance` is actually assigned and used by the base class (in PATCH and PUT) so I was just following that pattern. 

That said, there is no reason not to use `lambda` here, switch to using that.